### PR TITLE
rtorrent: add page

### PR DESCRIPTION
**Pull request by sasha42**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/rtorrent.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# rtorrent
+
+> Download torrents over the command line.
+
+- Add a torrent file or magnet to be downloaded:
+
+`rtorrent {{torrent or magnet}}`
```
Comment (agnivade) [inline pages/linux/rtorrent.md]: torrent_or_magnet_file

Comment (sasha42) [reply pages/linux/rtorrent.md]: It would be a torrent file or magnet link in that case.. should I change the text on line 7?

Comment (agnivade) [reply pages/linux/rtorrent.md]: Ah you are right. Let's keep it torrent_or_magnet then.

**Comment by sbrl** [comment]
Ping @sasha42

**Comment by sasha42** [comment]
I updated the description to torrent_or_magnet

**Comment by sbrl** [comment]
Thanks, @sasha42!

**Pull request merged**